### PR TITLE
Update repo

### PR DESCRIPTION
**Pull request by gabrielenava**
I think it is time to merge the WBM-class into master. But before doing this, it is necessary to verify the controllers are still working properly, and that's why I want to first merge this branch into WBM-class.

### CODE REVIEW

**Comment by gabrielenava** [comment]
@Ganimed do you agree with this merge? I worked mostly at the high level (matlab functions) so nothing is changed in your code

**Comment by Ganimed** [comment]
Try it, then we will see. Or better make a copy of the WBM-Class branch and merge it there.

**Comment by gabrielenava** [comment]
@ganimed if you prefer, I can merge `updateRepo` into master instead of your branch, considering that when I resolved the conficts I merged your commits into my branch so it is currently the most updated branch of the repo

File: controllers/torqueBalancing/init/initVisualizer.m [hunk 1]
```diff
@@ -21,9 +21,37 @@
 
 %% Robot simulator
 if CONFIG.visualize_robot_simulator == 1
+    % list of joints used in the visualizer
+    CONFIG.mdlLdr           = iDynTree.ModelLoader();
+    CONFIG.consideredJoints = iDynTree.StringVector();
     
-    references        = trajectoryGenerator(CONFIG.initForKinematics.xCoM,t,CONFIG);
-    CONFIG.figureCont = visualizeSimulation(t,chi,CONFIG,references);
+    CONFIG.consideredJoints.push_back('torso_pitch');
```
Comment (traversaro) [inline controllers/torqueBalancing/init/initVisualizer.m]: This should be automatically extracted by the used joint lists, but at the moment that is not exposed in mex-wholebodymodel, so I see why you had to hardcode this.

File: controllers/tools/visualization/visualizeSimulation_iDyntree.m [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+% get the joint position
+CONFIG.demux.baseOrientationType = 1;
+[~,qj,~,~] = stateDemux(chi',CONFIG);
+
+% visualization time
+init_time  = 1;
+end_time   = length(qj(1,:));
+
+% load the model from .urdf
+mdlLdr.loadReducedModelFromFile('model/model.urdf',consideredJoints);
```
Comment (traversaro) [inline controllers/tools/visualization/visualizeSimulation_iDyntree.m]: This should be exposed by the C++ to be sure that we are loading the same model.urdf of the underlying model.

Comment (traversaro) [inline controllers/tools/visualization/visualizeSimulation_iDyntree.m]: But for doing that we should first make sure that we install the meshes together with the urdf, something that at the moment we are not doing (see https://github.com/robotology-playground/icub-model-generator/issues/28 and https://github.com/robotology-playground/icub-model-generator/issues/33).

**Comment by gabrielenava** [comment]
updateRepo will be merged into master so no need for this pull request anymore

**Pull request closed without merging**